mod/zaudio/views/default/js/zaudio/boot.js

```diff
@@ -0,0 +1,16 @@
+define(function (require) {
+	var $ = require('jquery');
+
+	return {
```

Comment:
It might be easier for users to create these modules correctly if they were required to return objects and the booted module complained if unexpected keys were present. Otherwise a typo would cause code to silently not run.